Debugger/tests/System/BasicTest.php

```diff
@@ -72,7 +72,7 @@ public function testRegisterSetUseBreakpoint($transport)
         $this->assertNotEmpty($debuggee->id());
 
         // Set a breakpoint
-        $breakpoint = $debuggee->setBreakpoint('tests/system/Debugger/BasicTest.php', __LINE__);
+        $breakpoint = $debuggee->setBreakpoint('Debugger/tests/System/BasicTest.php', __LINE__);
```

Comment:
I think we should be able to remove the `Debugger` prefix. The source location resolver will find the file and will allow this test to pass when run from the root and from the `Debugger` folder

Reply:
Updated